src/node_api.cc

```diff
@@ -637,7 +637,8 @@ void napi_clear_last_error() {
   static_last_error.engine_reserved = nullptr;
 }
 
-const napi_extended_error_info* napi_get_last_error_info() {
+const napi_extended_error_info* napi_get_last_error_info(napi_env env) {
+  (void)env;
```

Comment:
Assume this is to suppress unused formal argument warnings? Suppose what we want is actually to use something like ```CHECK_ARG(env);``` instead. That would require ```napi_get_last_error_info()``` to return ```napi_status```, though...